### ISSUE TITLE
Creating changesets without comment

### ISSUE THREAD

**Issue by naoliv** [mapsme/omim#2998]
It seems that somehow maps.me is creating changesets without any comment, like this one: https://www.openstreetmap.org/changeset/38772063

Shouldn't they all have some kind of comment?

**Comment by deathbaba** [comment]
Yes, they should. It's a bug, connection is terminated before comment is sent.

CC @burivuh
